Example/patches/metro-inspector-proxy+0.76.8.patch

```diff
@@ -1,177 +0,0 @@
-diff --git a/node_modules/metro-inspector-proxy/src/Device.js b/node_modules/metro-inspector-proxy/src/Device.js
```

Comment:
Does debugging with Chrome DevTools still work fine if this patch is removed?

Reply:
There is no such package in `node_modules` right now and I didn't check. @piaskowyk declared that you will take care of it later, am I right?